client/src/components/CalendarWidget/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import Calendar from "react-calendar";
```

Comment:
I am unable to resolve this module when I run `npm start`. Was `react-calendar` supposed to be included in the `package.json`?